PowervRA/Functions/Public/config-service/Connect-vRAVAMI.psm1

```diff
@@ -0,0 +1,130 @@
+function Connect-vRAVAMI {
+<#
+    .SYNOPSIS
+    Connect to a vRA Appliance
+    
+    .DESCRIPTION
+    Connect to a vRA Appliance and generate a connection object with Servername, Token etc
+    
+    .PARAMETER Appliance
```

Comment:
Suggestion: Maybe change Appliance to Server to be consistent with Connect-vRAServer? Happy either way though.